src/tools/x/Cargo.lock

```diff
@@ -0,0 +1,5 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+[[package]]
+name = "bootstrap"
+version = "1.0.2"
```

Comment:
This is left over from experimenting I think, `Cargo.toml` has no such dependency.

Reply:
Fixed